flink-clients/src/main/java/org/apache/flink/client/deployment/AbstractJobClusterExecutor.java

```diff
@@ -0,0 +1,4 @@
+package org.apache.flink.client.deployment;
+
+public class AbstractJobClusterExecutor {
```

Comment:
There are some empty classes being added in this commit. They are probably not needed and are changed later.

Reply:
You are right, I will squash upon merging.